ci/release/apply_wheel_modifications.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Usage: bash apply_wheel_modifications.sh <new_version> <cuda_suffix>
+
+VERSION=${1}
+CUDA_SUFFIX=${2}
+
+# __init__.py updates
+sed -i "s/__version__ = .*/__version__ = \"${VERSION}\"/g" python/cugraph/cugraph/__init__.py
+sed -i "s/__version__ = .*/__version__ = \"${VERSION}\"/g" python/cugraph-dgl/cugraph-dgl/__init__.py
+sed -i "s/__version__ = .*/__version__ = \"${VERSION}\"/g" python/cugraph-pyg/cugraph-pyg/__init__.py
+sed -i "s/__version__ = .*/__version__ = \"${VERSION}\"/g" python/cugraph-service/client/cugraph_service_client/__init__.py
+sed -i "s/__version__ = .*/__version__ = \"${VERSION}\"/g" python/cugraph-service/server/cugraph_service_server/__init__.py
+sed -i "s/__version__ = .*/__version__ = \"${VERSION}\"/g" python/pylibcugraph/pylibcugraph/__init__.py
```

Comment:
```suggestion
sed -i "s/__version__ = .*/__version__ = \"${VERSION}\"/g" \
  python/cugraph/cugraph/__init__.py \
  python/cugraph-dgl/cugraph-dgl/__init__.py \
  python/cugraph-pyg/cugraph-pyg/__init__.py \
  python/cugraph-service/client/cugraph_service_client/__init__.py \
  python/cugraph-service/server/cugraph_service_server/__init__.py \
  python/pylibcugraph/pylibcugraph/__init__.py
```

`sed` can be run against multiple files.

Should we DRY up these commands like this?

Reply:
Yeah we can, I just hadn't initially because it doesn't work in update-version.sh due to the sed_runner.